Android.mk

```diff
@@ -114,6 +114,13 @@ endif
 
 ifeq ($(KERNELFLINGER_SUPPORT_USB_STORAGE),true)
     KERNELFLINGER_CFLAGS += -DUSB_STORAGE
+    ifeq ($(KERNELFLINGER_SUPPORT_LIVE_BOOT),true)
+        ifneq ($(TARGET_BUILD_VARIANT),user)
+            KERNELFLINGER_CFLAGS += -DLIVE_BOOT
```

Comment:
why we need define new macro.  The purpose is to make boot from removable device only work on userdebug build. Why not use macro USERDEBUG directly

Reply:
We still support use USB disk as main storage, so live boot should be an option.